docs/rfcs/0013-rfc-process-updates.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: RFC process updates
+permalink: /docs/rfcs/0013-rfc-process-updates
+---
+
+* RFC PR: https://github.com/mozilla-mobile/firefox-android/pull/5681
+* Start date: 2024-02-20 
+* End date: 2024-03-22 
+* Stakeholders: @jonalmeida, @boek
+
+## Summary
+
+Clarify requirements for RFCs by introducing a README and a template.
+
+## Motivation
+
+- The scale of the Firefox Android team has grown substantially since the RFC process was first introduced. 
+- More external teams are using and contributing to Android Components. 
+- Approval or rejection of a proposal has required periods of time that can slow implementation.
```

Comment:
The approval or rejection time is set by the author already. The RFC is just a pull request that goes through the same process as a code review: if you have an approval, you can merge that work. Additional reviewers are welcome to add feedback that is up to the author to consider if they want to address it in the current PR, future work, or not applicable.

I'm not sure this is a valid motivation. What is the difference that we are adding here?

Reply:
The intent here is clarity and centralization of information. Setting a deadline in a different communication channel divorces that information from the proposal if the proposal is shared directly, for example.

My goal is to make these easier to write for people unfamiliar with the process, and codifying things like this I think will assist in knowledge transfer - future authors will have less reliance on offline (and usually hidden from public/history) mentorship.